pkg/kube/inject/inject_test.go

```diff
@@ -587,17 +587,17 @@ func TestCleanProxyConfig(t *testing.T) {
 		{
 			"default",
 			mesh.DefaultProxyConfig(),
-			`{}`,
+			`{"controlPlaneAuthPolicy":"MUTUAL_TLS"}`,
```

Comment:
The whole point of the test is this is supposed to be empty. Can you update this?

Reply:
Basically the intent is for a default proxy config (99% of users) we should not pollute the pod config

Reply:
> The whole point of the test is this is supposed to be empty. Can you update this?

Thanks @howardjohn , Fixed in https://github.com/istio/istio/pull/25460